### PR TITLE
Auto scrolling

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

File: static/js/ScreenManager.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ class ScreenManager {
     }
 
     setHandlers() {
+        console.log(this.terminalButtons);
```
Comment (jaanonim) [inline static/js/ScreenManager.js:37]: console.log

**Review by jaanonim — APPROVED** [review]
LGTM

**Review by Adron4444 — APPROVED** [review]
LGTM

**Review by mwawrzen — APPROVED** [review]
LGTM

**Pull request merged**